kpatch-build/kpatch-build

```diff
@@ -88,6 +89,17 @@ logger() {
 	fi
 }
 
+save_env() {
+	export -p | grep -wve 'OLDPWD=' -wve 'PWD=' > "$ENVFILE"
```

Comment:
Now that the saved .env file is up in $CACHEDIR/tmp and not in the patch directory, we need to filter out the `{OLD,}PWD` variables to avoid some bash environment weirdness when sourcing the file.

Reply:
> I don't think we need to specify `-w` and `-v` twice...
Yes, applying those flags once, separately from the `-e` expressions looks cleaner.